components/plugins/lints/sorter.rs

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use rustc::lint::{Context, LintPass, LintArray};
+use std::cmp::Ordering;
+use std::fmt::Display;
+use syntax::ast::{Mod, Item_, PathListItem_, ViewPath_};
+use syntax::codemap::{Pos, BytePos, Span};
+use syntax::print::pprust::path_to_string;
+
+declare_lint!(UNSORTED_DECLARATIONS, Deny,
```

Comment:
This should be `Warn`. `Deny` is going to be very annoying when you're fiddling with imports temporarily before checking in.